services/ui-src/src/forms/mcpar/bpc/bpc.json

```
@@ -1,18 +1,50 @@
 {
   "path": "/mcpar/state-level-indicators/program-characteristics",
   "intro": {
-    "section": "B: State-Level Indicators",
-    "subsection": "I: Program Characteristics"
+    "section": "Section B: State-Level Indicators",
+    "subsection": "I. Program Characteristics and Enrollment"
   },
   "form": {
     "id": "bpc",
     "fields": [
       {
         "id": "bpc-1",
-        "type": "text",
+        "type": "number",
         "props": {
-          "label": "Temporary placeholder field",
-          "disabled": true
+          "label": "B.I.1 Statewide medicaid enrollment",
+          "hint": [
+            {
+              "type": "text",
+              "as": "p",
+              "content": "Enter the total number of individuals enrolled in Medicaid as of the first day of the last month of the reporting year."
+            },
+            {
+              "type": "text",
+              "as": "p",
+              "content": "Include all FFS and managed care enrollees, and count each person only once, regardless of the delivery system(s) in which they are enrolled."
+            }
```

Nit: We could make these ones with the html parser like this:
```suggestion
            {
              "type": "html",
              "content": "<p>Enter the total number of individuals enrolled in Medicaid as of the first day of the last month of the reporting year.</p><p>Include all FFS and managed care enrollees, and count each person only once, regardless of the delivery system(s) in which they are enrolled.</p>"
            },
```
Totally optional.